app/jobs/address_proof_job.rb

```diff
@@ -0,0 +1,25 @@
+class AddressProofJob
```

Comment:
Wait we don't need this class right? We should just be able to use `IdentityIdpFunctions::ProofAddress` as a drop-in? Or maybe we need to make the mock version? Since loading the real one in tests would cause lexisnexis to be loaded

Reply:
Ah ok, I thought the mock case was still going to live in the IDP and the lambda was strictly calls being made with the vendor.

Reply:
If so, then we should make this clearly a Mock job or something

Reply:
Like we could just hardcode the mock proofer, remove the HTTP callback bit